docs/paper/admin/how-to/get-to-vanilla.mdx

```diff
@@ -0,0 +1,38 @@
+---
+slug: /vanilla
+description: This page lists all changes that result in a different experience than vanilla.
+---
+
+# Hot to get a vanilla experience
+
+import React from 'react';
+import ConfigDocBlock from '@site/src/components/config/ConfigDocBlock';
+import VanillaWorldDefaults from '!!raw-loader!@site/config-specs/paper/vanilla/paper-world-defaults.yml'
+import VanillaGlobalDefaults from '!!raw-loader!@site/config-specs/paper/vanilla/paper-global.yml'
+import SpigotChanges from '!!raw-loader!@site/config-specs/paper/vanilla/spigot.yml'
+
+Due to the way how the Bukkit platform is built and because it fixes vanilla bugs, the gameplay experience between Vanilla and Paper can lead to inconsistencies.
```

Comment:
iirc it's not exactly the fault of the platform, just that craftbukkit has some old cruft that messes with behavior

Reply:
I think it's easier to understand this way. If someone has a better suggestion I can change it.

Reply:
```suggestion
Due to the way the Bukkit API is implemented in Paper, the gameplay experience between Vanilla and Paper can have inconsistencies. This can be furthered by efforts to fix bugs present in Vanilla Minecraft.
```